redis-test/Cargo.toml

```diff
@@ -13,7 +13,7 @@ rust-version = "1.63"
 bench = false
 
 [dependencies]
-redis = { version = "0.23.2", path = "../redis" }
+redis = { version = "0.23.3", path = "../redis" }
```

Comment:
minor: we probably don't need the version tag here and below, because we're fetching the dependency from a path, so cargo has only a single version to choose from.

Reply:
Yep that is correct. Let's get it next time.